pkg/trivyk8s/deprecatedapi.go

```diff
@@ -0,0 +1,32 @@
+package trivyk8s
```

Comment:
can we add a comment saying this file shoulnd't be edited since it's generated by the makefile? (alternatively, use go:generate)

Reply:
add a comment https://github.com/aquasecurity/trivy-kubernetes/pull/458/commits/a4bcb4ed3f54e8056868e0ed7f34b2db2d679f35